doc/running-native.md

```diff
@@ -1,27 +1,27 @@
 
-# Installing natively
+# Running natively
 
 This page describes the procedure to install and run PrairieLearn without any use of Docker. This means that PrairieLearn is running fully natively on the local OS. This installation method is tested and supported on MacOS and Linux, but not on Windows. It should also work on Windows, but it is not tested.
 
-* Install the pre-requisites:
+1. Install the pre-requisites:
 
     * [Node.js](http://nodejs.org/) version  or higher
     * [npm](https://npmjs.org/) (included with Node.js on Windows)
     * [PostgreSQL](https://www.postgresql.org) version 9.6 or higher
     * [Python 3](https://www.python.org) version 3.6 or higher
     * command-line git or [GitHub Desktop](https://desktop.github.com)
 
-On OS X these can be installed with [MacPorts](http://www.macports.org/) or [Homebrew](http://brew.sh/). On Linux these should all be standard packages from the OS distrbution.
+On OS X these can be installed with [MacPorts](http://www.macports.org/) or [Homebrew](http://brew.sh/). On Linux these should all be standard packages from the OS distribution.
 
 Note that with MacPorts you need to select the active version of PostgreSQL, for example `port select postgresql postgresql96`.
```

Comment:
we can probably delete this now -- anyone reading this should know how to use their package manager

Reply:
You'd be surprised how many people still don't know about HomeBrew 🙂 I'd rather leave this in for now